packages/EasyErrorHandler/src/Bridge/Laravel/config/easy-error-handler.php

```diff
@@ -7,7 +7,10 @@
     /**
      * Use extended error response with exception message, trace, etc.
      */
-    'use_extended_response' => \env('EASY_ERROR_HANDLER_USE_EXTENDED_RESPONSE', false),
+    'use_extended_response' => \filter_var(
```

Comment:
Please, cast to boolean in the `isExtendedResponse` method instead. This package uses this approach.

Reply:
Done.

Reply:
I don't understand why do we have to do all that? The laravel `env()` method is already taking care of casting boolean values. Isn't that working?

```php
// vendor/illuminate/support/Env.php

switch (strtolower($value)) {
	case 'true':
	case '(true)':
		return true;
	case 'false':
	case '(false)':
		return false;
	case 'empty':
	case '(empty)':
		return '';
	case 'null':
	case '(null)':
		return;
}
```